gstreamer-video/src/video_info.rs

```diff
@@ -166,6 +166,14 @@ impl fmt::Display for ::VideoColorimetry {
 
 pub struct VideoInfo(pub(crate) ffi::GstVideoInfo);
 
+impl fmt::Debug for VideoInfo {
```

Comment:
This should probably be a separate commit

Reply:
Done.